config/locales/views/en.yml

```diff
@@ -325,46 +325,59 @@ en:
   abuse_reports:
     new:
       heading:
-        page_title: Report Abuse
+        page_title: 'Policy Questions & Abuse Reports'
       purview:
-        about: The Abuse committee looks after all of your concerns regarding content and user behavior which breach the Archive %{tos_link}. For technical help, please %{support_link}.
```

Comment:
This line also shows up in one of the cucumber tests: https://github.com/otwcode/otwarchive/blob/master/features/other_a/abuse_report.feature#L36. Personally, I think it would be fine to just remove it